systems/controllers/osc/operational_space_control.cc

```diff
@@ -318,52 +347,21 @@ void OperationalSpaceControl::Build() {
               .get();
     }
   }
-  // 4. Friction constraint (approximated friction cone)
   if (!all_contacts_.empty()) {
     VectorXd mu_neg1(2);
     VectorXd mu_1(2);
     VectorXd one(1);
-    mu_neg1 << mu_, -1;
-    mu_1 << mu_, 1;
-    one << 1;
+    MatrixXd A = MatrixXd(5, kSpaceDim);
```

Comment:
Can we name this A_fcone or something and include a comment for how many faces we are using? I assume a pyramid?